.github/workflows/wf-build-release.yml

```
@@ -1,195 +1,69 @@
-      run: dotnet nuget push "**/*.nupkg" --no-symbols --skip-duplicate --api-key $env:GH_PKG_SEC --source https://nuget.pkg.github.com/codebude/index.json
-      
-    
-  clean:
-    needs: [build, test, pack-push-release]
-    if: always()
-    runs-on: windows-latest
-    steps:
-    - name: Delete artifacts
-      uses: GeekyEggo/delete-artifact@v5
+    - name: Upload nuget packages as release artifacts
+      uses: actions/github-script@v4
```

actionlint (1.7.7)

the runner of "actions/github-script@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)